client-java/instrumentation/pom.xml

```diff
@@ -97,6 +97,14 @@
             <artifactId>jackson-annotations</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
check it if these two dependencies are only applied for `test` scope.
if not, you need to add the two dependencies with the plugin
`<groupId>org.apache.maven.plugins</groupId>
 <artifactId>maven-shade-plugin</artifactId>`

see below of this `pom.xml`